labonneboite/common/models/office.py

```diff
@@ -115,20 +118,39 @@ class Office(FinalOfficeMixin, CRUDMixin, Base):
     def __unicode__(self):
         return u"%s - %s" % (self.siret, self.name)
 
-    def as_json(self, rome_code=None, distance=None, zipcode=None, extra_query_string=None):
+    def as_json(self,
+            rome_codes=None,
+            hiring_type=hiring_type_util.DEFAULT,
+            distance=None,
+            zipcode=None,
+            extra_query_string=None,
+        ):
         """
-        `rome_code`: optional parameter, used only in case of being in the context
-        of a search by ROME code.
-        Without the context of a ROME code, the general purpose score of the office
+        `rome_codes`: optional parameter, used only in case of being in the context
+        of a search by ROME codes (single rome or multi rome).
+        Without the context of ROME codes, the general purpose score of the office
         is returned.
-        With the context of a ROME code, the score returned is adjusted to the ROME code,
+        With the context of ROME codes, the score returned is adjusted to the ROME code,
         and the URL of the company page is also adjusted to keep the same context.
         Main case is results returned by an API search. The scores and URLs embedded
         in the company objects should be adjusted to the ROME code context.
 
+        `hiring_type`: is needed along rome_codes to compute the right corresponding score.
```

Comment:
Please specify allowed values for this argument.

Reply:
ok mais je vais checker s'il n'y pas des cas où elle doit être appelée indépendement

Reply:
done